pkg/channel/http/http_channel.go

```diff
@@ -75,12 +77,12 @@ func CreateLocalChannel(port, maxConcurrency int, pipeline httpMiddleware.Pipeli
 
 	c := &Channel{
 		pipeline: pipeline,
-		client: &fasthttp.Client{
-			MaxConnsPerHost:           1000000,
-			MaxIdemponentCallAttempts: 0,
-			MaxResponseBodySize:       maxRequestBodySize * 1024 * 1024,
-			ReadBufferSize:            readBufferSize * 1024,
-			DisablePathNormalizing:    true,
+		// We cannot use fasthttp here because of lack of streaming support
+		client: &http.Client{
```

Comment:
@ItalyPaleAle I believe that setting `MaxConnsPerHost` on the http.Client will improve performance by a wide margin (you need to create a `DefaultTransport`).

Reply:
Thanks, this sounds like solid advice